src/lib/wpcom-factory.ts

```diff
@@ -0,0 +1,9 @@
+import wpcomModule from 'wpcom';
+
+// Normalize for both ESM (Vite/browser) and CJS (Electron IPC) contexts.
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const wpcomFactory: any =
+	typeof wpcomModule === 'function' ? wpcomModule : ( wpcomModule as any ).default ?? wpcomModule;
```

Comment:
This second fallback could be omitted.

```suggestion
	typeof wpcomModule === 'function' ? wpcomModule : ( wpcomModule as any ).default;
```

Reply:
Removed in 5ef78a5802b0c0b3b0ee84627bcfb9c8ba58a83c